src/bundle/Core/DependencyInjection/Configuration/RepositoryConfigParser.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+/**
+ * @copyright Copyright (C) Ibexa AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+declare(strict_types=1);
+
+namespace Ibexa\Bundle\Core\DependencyInjection\Configuration;
+
+use eZ\Publish\Core\Base\Exceptions\InvalidArgumentType;
+use Symfony\Component\Config\Definition\Builder\NodeBuilder;
+
+final class RepositoryConfigParser implements RepositoryConfigParserInterface
+{
+    /** @var \Ibexa\Bundle\Core\DependencyInjection\Configuration\RepositoryConfigParserInterface[] */
+    private $configParsers;
+
+    /**
+     * @param \Ibexa\Bundle\Core\DependencyInjection\Configuration\RepositoryConfigParserInterface[] $configParsers
+     */
+    public function __construct(array $configParsers = [])
```

Comment:
Optional: Argument and property can be made `iterable`. We're not using functionalities specific to arrays anywhere.

Reply:
Fixed in https://github.com/ezsystems/ezplatform-kernel/pull/218/commits/69ff21ef0a40cbf323d7ef379f1bcaaffdca4f64